code/API_definitions/qos-profiles.yaml

```
@@ -339,6 +339,20 @@ components:
           minimum: 1
           maximum: 10
           example: 3
+        l4sQueueType:
+          type: string
+          enum:
+            - non-l4s-queue
+            - l4s-queue
+            - mixed-queue
+          description: Specifies the type of queue for L4S traffic management
+        differentiatedServicesCodepoint:
+          type: array
+          items:
+            type: integer
+          minItems: 0
+          maxItems: 64
+          description: A list of unique DSCP values from 0-63
```

```suggestion
          description: A list of unique DSCP values from 0-63, where each decimal value is mapped to the 6 bit DSCP value.  For example `0` would be `000000` and `8` would be `001000`.  Where 0 and 8 would map to `CS0` and `CS1` [RFC 2474](https://www.rfc-editor.org/rfc/rfc2474).
```
We should also add the mapping to some alternate standards as well.

---

I added some additional information - please see my suggestion below